### PR TITLE
[DEVOPS-413] development clusters

### PR DESCRIPTION
**Pull request by domenkozar**

### CODE REVIEW

**Comment by domenkozar** [comment]
Changes for development cluster seem good, but we should leave out cardano-sl bump and configuration.yaml unless we properly merge those upstream.

File: topology-development.yaml [hunk 1]
```diff
@@ -3,42 +3,52 @@ nodes:
...
     type: relay
     region: ap-southeast-1
     zone:   ap-southeast-1b
-    static-routes: [["a1"]]
-    addr: r1.cardano
+    static-routes: [["a1"], ["b1"]]
+    host: r1.cardano
     port: 3000
+    kademlia: false
   'r2':
```
Comment (domenkozar) [inline topology-development.yaml]: We only need one relay really

File: cardano-sl-src.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "url": "https://github.com/input-output-hk/cardano-sl",
     "fetchSubmodules": "true",
-    "sha256": "0hrfha4xsvdxhfnkpjc3bclijsixwibnckvvb9mbf8kc563dv61f",
-    "rev": "f2e1da159543b439f30ecef2d288d4661d6765a5"
+    "sha256": "0a480p4wcjj8hi7bcgkv85jy94ad59isfg7lfnlk8qsv0q2bj1wh",
+    "rev": "7f5aff98f39ccf50ec3f5c98638e93535f16021c"
```
Comment (domenkozar) [inline cardano-sl-src.json]: should be removed

File: configuration.yaml.tmpl [hunk 3]
```diff
@@ -14955,3 +14955,26 @@ testnet_staging_wallet_macos64: &testnet_staging_wallet_macos64
     <<: *mainnet_base_update
     applicationName: csl-daedalus
     systemTag: macos64
+
+##############################################################################
```
Comment (domenkozar) [inline configuration.yaml.tmpl]: to be removed

**Comment by domenkozar** [comment]
I'll make the necessary changes here

**Comment by deepfire** [comment]
@domenkozar, I think we also need a configuration.yaml update, let me check..

**Comment by domenkozar** [comment]
@deepfire we are going to use `devnet` for development clusters when it's all ready, I've removed the configuration.yaml changes that were specific to the branch.

**Comment by deepfire** [comment]
@domenkozar, I think @gromakovsky already has a `devnet` configuration key prepared somewhere.. we might include it here as well -- and update `iohk-ops` to use it.

**Comment by gromakovsky** [comment]
Right, `devnet` is in master.

**Pull request merged**